dynamic_island_watermark/lib/dynamic_island_watermark.dart

```diff
@@ -132,6 +139,22 @@ class _IslandWatermarkState extends State<_IslandWatermark> {
       _ => null,
     };
   }
+
+  bool _isPortrait() {
+    final mediaQuery = MediaQuery.maybeOf(context);
+    if (mediaQuery != null) {
+      return mediaQuery.orientation == Orientation.portrait;
+    }
```

Comment:
Why don't you use `MediaQuery.maybeOrientationOf()`?

Reply:
Uh I didn't know about this, that's super nice, thanks a lot for pointing that out ✨